packages/env-tests/src/env.ts

```diff
@@ -0,0 +1,60 @@
+import * as dotenv from 'dotenv'
+import { existsSync } from 'fs'
+import path from 'path'
+
+if (process.env.CONFIG) {
+  dotenv.config({ path: process.env.CONFIG })
+}
+
+export function fetchEnv(name: string): string {
+  if (process.env[name] === undefined || process.env[name] === '') {
+    console.error(`ENV var '${name}' was not defined`)
+    throw new Error(`ENV var '${name}' was not defined`)
+  }
+  return process.env[name] as string
+}
+
+export function fetchEnvOrDefault(name: string, defaultValue: string): string {
+  return process.env[name] === undefined || process.env[name] === ''
+    ? defaultValue
+    : (process.env[name] as string)
+}
+
+export function isYes(value: string) {
+  switch (value.toLowerCase().trim()) {
+    case '1':
+    case 'y':
+    case 'yes':
+    case 't':
+    case 'true':
```

Comment:
I've seen an NPM package that accepts all yes like things, yes, yeah, y, oui, ja, etc